R/module_teal_data.R

```diff
@@ -129,11 +129,11 @@ srv_validate_reactive_teal_data <- function(id, # nolint: object_length
     # there is an empty reactive cycle on `init` and `data_rv` has `shiny.silent.error` class
     srv_validate_error("silent_error", data, validate_shiny_silent_error)
     srv_check_class_teal_data("class_teal_data", data)
-    srv_check_shiny_warnings("shiny_warnings", data, modules)
+    srv_check_module_datanames("shiny_warnings", data, modules)
```

Comment:
a minor comment:
if this is meant to raise error then maybe "assert" instead of "check" (using checkmate nomenclature)
this could be done separately

Reply:
This doesn't stop execution so `assert` doesn't quite fit. It's a gentle warning to push app developers to correct.

Reply:
ok thanks for checking
then my comment is irrelevant - closing